src/features/people/popularPeopleSlice.js

```diff
@@ -3,34 +3,29 @@ import { createSlice } from "@reduxjs/toolkit";
 const popularPeopleSlice = createSlice({
   name: "popularPeople",
   initialState: {
-    page: 1,
-    totalPages: null,
```

Comment:
Zgadzam się zdecydowanie, że to już nie będzie potrzebne ;) Po prostu na etapie tworzenia wolałem to dodać i zobaczyć jak to później będzie, no i wyszło w praniu, że niepotrzebne ;)